projects/vdk-plugins/vdk-jobs-troubleshooting/.plugin-ci.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+image: "python:3.7"
+
+.build-vdk-jobs-troubleshooting:
+  variables:
+    PLUGIN_NAME: vdk-jobs-troubleshooting
+  extends: .build-plugin
+
+build-py37-vdk-jobs-troubleshooting:
+  extends: .build-vdk-jobs-troubleshooting
+  image: "python:3.7"
+
+
+build-py38-vdk-jobs-troubleshooting:
+  extends: .build-vdk-jobs-troubleshooting
+  image: "python:3.8"
+
+
+build-py39-vdk-jobs-troubleshooting:
+  extends: .build-vdk-jobs-troubleshooting
+  image: "python:3.9"
+
+build-py310-vdk-jobs-troubleshooting:
+  extends: .build-vdk-jobs-troubleshooting
+  image: "python:3.10"
+
+build-py311-vdk-jobs-troubleshooting:
+  extends: .build-vdk-jobs-troubleshooting
+  image: "python:3.11"
+
+build-vdk-jobs-troubleshooting-on-vdk-core-release:
```

Comment:
Comment out this for now. You will enable it when the plugin is more mature. Plugin this early in development should not block vdk-core CICD

Reply:
Good point. Commented out, thanks